pkg/webhook/webhook.go

```diff
@@ -60,6 +62,10 @@ func (pwh *pravegaWebhookHandler) Handle(ctx context.Context, req admissiontypes
 	}
 	copy := pravega.DeepCopy()
 
+	if err := pwh.clusterIsAvailable(ctx, copy); err != nil {
+		return admission.ErrorResponse(http.StatusBadRequest, err)
```

Comment:
I believe that, in this case, we should return a server error (5xx) instead of a user error (4xx). `BadRequest` is telling the user that the request is malformed, which is not the case, it's just that the cluster status does not allow this type of request temporarily.

Reply:
Got it!